ui/imports/shared/popups/ConfirmationDialog.qml

```diff
@@ -11,7 +11,7 @@ StatusModal {
     id: confirmationDialog
     anchors.centerIn: parent
 
-    property Popup parentPopup
+    property var parentPopup
```

Comment:
Do we need this property at all? It's not used in this component.

Reply:
Good point and I'll most probably gonna remove it anyway as part of #7360

Reply:
ok, so let's merge that and and put on hold any further changes related to old profile modals :)